<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QMainWindow" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>379</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>379</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="main_pic.qrc">
      <normaloff>:/mainpic/pic/旧刻.png</normaloff>:/mainpic/pic/旧刻.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="backbutton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="main_pic.qrc">
      <normaloff>:/mainpic/pic/fanhui.png</normaloff>:/mainpic/pic/fanhui.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="main_pic.qrc">
      <normaloff>:/mainpic/pic/作石.png</normaloff>:/mainpic/pic/作石.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="main_pic.qrc"/>
 </resources>
 <connections/>
</ui>
